frontend/.eslintrc.js

```diff
@@ -0,0 +1,18 @@
+module.exports = {
+  settings: {
+    react: {
+      version: 'detect'
+    }
+  },
+  env: {
+    browser: true,
+    node: true,
+    es6: true
+  },
+  plugins: ['react'],
+  rules: {
+    'react/prop-types': 0
```

Comment:
i think 0 means ignore, right? then why? i think it is good to define prop-types for all components, it help with the auto complete and other stuff

Reply:
> i think 0 means ignore, right? 
> 
> then why? i think it is good to define prop-types for all components, it help with the auto complete and other stuff.
> 
> regardless the decision whether or not to disable this rule, please use the descriptive level version "off", "warn" or "error" instead of 0,1 or 2
> 
> [read more](https://eslint.org/docs/user-guide/configuring) 
> 

Will im not used to defining proptypes in javascript so lets work on easy mode for now and in the future we might use typescript